### ISSUE TITLE
When using for 20+ servers, the front-end hangs often.

### ISSUE THREAD

**Issue by arthurstreeter** [tiagocoutinho/multivisor#67]
It becomes very unresponsive. This is probably a very unique situation, but upon having many servers/processes being reported the entire front-end no matter the browser becomes terribly unresponsive.
Has anyone figured out a solution for this? I'm not the greatest at Vue.

I know I could possibly  divvy up my multivisor config file and run separate instances on different ports, but that defeats the purpose of this great tool.

**Comment by tiagocoutinho** [comment]
How many supervisor instances and how many processes are you trying?

Should I be able to reproduce the behaviour with only one supervisor instance with a lot of processes?

> I know I could possibly divvy up my multivisor config file and run separate instances on different ports, but that defeats the purpose of this great tool.

Agree. If multivisor is not scaling well we should tackle the problem. I thank you for reporting this. It helps multivisor be a better tool

**Comment by arthurstreeter** [comment]
So I'm currently running about 30+ supervisor instances out of 80. Each one has about 6-10 processes that are being watched. There are times when a server may be acting up and a particular process is ended/started over and over again which causes the front-end to have a pop-up on the bottom right corner of the screen giving me an update each time it's instance's status changes. I'm not sure if that has anything to do with it. I do appreciate those alerts, however, but I would understand if they needed to be toned down a bit when a lot of supervisors are present.

When I had all 80+ servers in the config file, the page was rendered near useless as even the act of scrolling up and down the page was a hassle as it would take time to load the processes from each hosted group.

Let me see if I can replicate the issue with a ton of repeated processes on the same supervisor instance.

**Comment by arthurstreeter** [comment]
Yes, I was able to replicate the same issue by having one supervisor instance with 6 processes repeated 40+ times (each time with a different supervisor name, but same url).
